Added two new methods:

rahuljiresal opened the pull request:
I wanted a persistent message that tells users if a newer version is available. For example, in the Settings page of my app, I show the current version of the app. I wanted to show if a newer version is available, and if yes, have a non-modal option to go to the AppStore. Something like the image below.

![ios simulator screen shot may 24 2015 9 54 30 pm](https://cloud.githubusercontent.com/assets/216346/7792051/3958102e-0260-11e5-8efb-7ef1b7178a41.png)

Current implementation of Harpy didn't allow me that. Hence I added two new methods. Maybe others will find them useful too?

1. [[Harpy sharedInstance] newVersionAvailable] - Returns YES/NO boolean value to tell if a new version is available. Uses a stored value from NSUserDefaults.
2. [[Harpy sharedInstance] launchAppStore] - Launches the AppStore for the app.

ArtSabintsev commented:
Hi @rahuljiresal 

Thanks for this solution. I actually have solved this problem in the sister project, [Siren](http://github.com/ArtSabintsev/Siren). I solved it by adding a delegate method. I haven't gotten around to doing it here just yet, but will try to do it ASAP. I'll ping you once it's done.

I very much appreciate your time and effort and will use this as fuel to get this done sooner rather than later. I'll close this issue for now (without accepting the PR), but will respond to this issue down the road when I implement the Siren-style delegate method solution.

Thanks!

rahuljiresal commented:
Hey @ArtSabintsev 

I recently had to deploy an app for enterprise distribution. So wanted to check against he enterprise pList file if a new version is available. 

Added a public method.
`[[Harpy sharedInstance] setPListURL:@"http://url"]`

Added a private method.
`- (void)performEnterpriseVersionCheck `

The commit is ready to be pulled and merged if you want to.

ArtSabintsev commented:
Hi @rahuljiresal,

Are enterprise plist files based on a common standard? To my knowledge, they do not. If they do, could you link me to an Apple document that defines what one of these plist looks like?

Thanks.

rahuljiresal commented:
Xcode used to generate plist files with the "Save for Enterprise" option while making an IPA. Not anymore since Xcode 6. So, unfortunately, there are no docs. However, some people have copied their older plist files and used those. I am doing that too. Here's a sample - http://stackoverflow.com/a/25916806/1314292

ArtSabintsev commented:
Ah, since Xcode 6 no longer supports it out of the box, I'm inclined to not add the feature. I would say the best thing for you to do going forward is to use a fork of Harpy for your own needs. I only say that because parsing a plist file is different than parsing the JSOn result from the iTunes API, and adds extra complexity for a solution only a very small handful of people will make use of.

rahuljiresal commented:
Okay, sounds good!
Good to close this one now?

ArtSabintsev commented:
Yes
